### ISSUE TITLE
Toasts aren't announced in Firefox + NVDA on Windows

### ISSUE THREAD

**Issue by paulGeoghegan** [shoelace-style/webawesome#2282]
Hi there, I was checking out Toasts again now that they are fixed as of 3.5 but actually they are not announced at all.
I'm on Windows using FF and NVDA.

**Comment by claviska** [comment]
Any chance you can you link to a library or two that announces toasts correctly on your combination of devices? Thanks!

**Comment by claviska** [comment]
OK, this has been fixed in the Pro repo and follows the same pattern as Radix: a persistent, shared live region is used and each toast is announced as a separate node which is removed shortly afterwards.

(For maintainers: https://github.com/shoelace-style/webawesome-pro/pull/206)
